Applets/UI/MainScreen/CellView.swift

```diff
@@ -8,46 +8,60 @@
 import SwiftUI
 
 struct CellView: View {
-
+    
     @ObservedObject var model: Goal
-
+    
     var body: some View {
-            ZStack {
-                if model.isEnabled {
-                    if model.percentOfDone < 100 {
-                        HStack(alignment: .lastTextBaseline) {
-                            Text("\(model.percentOfDone)")
-                                .font(.system(size: 100).bold().weight(.black))
-                                .padding(-10)
-                            Text("%")
-                                .font(.system(size: 40).bold().weight(.black))
-                                .padding(5)
-                        }
-                        .frame(width: 370, height: 100, alignment: .trailing)
-                        .foregroundColor(.white.opacity(0.30))
-                    } else {
-                        Text("DONE")
-                            .font(.system(size: 120).bold().weight(.black))
-                            .foregroundColor(.white.opacity(0.30))
+        ZStack(alignment: .center) {
+            VStack(alignment: .leading) {
+                Text(model.title ?? "")
+                    .font(.system(size: 20).bold())
+                    .frame(maxWidth: .infinity, alignment: .topLeading)
+                    .padding(.leading, 15)
+                    .padding(.top, 25)
+                    .padding(1)
+                    .foregroundColor(model.isEnabled ? .black : .gray)
+                
+                Text(model.goalDescription ?? "")
+                    .frame(maxWidth: .infinity, alignment: .topLeading)
+                    .padding(.leading, 15)
+                    .foregroundColor(model.isEnabled ? .gray : .gray.opacity(0.50))
+                
+                if model.fractionOfDone < 1 {
+                    ZStack {
+                        Rectangle()
+                            .frame(width: 330, height: 3)
+                            .foregroundColor(.clear)
+                            .modifier(FilledCellModifier(percentOfDone: model.fractionOfDone))
+                            .position(x: 182, y: 20)
```

Comment:
To make it correct for all kind of screen width we need to change this from absolute values to relative on screen width. Maybe now we can merge but consider it for the future